src/istioctl/istioctl.go

```diff
@@ -203,84 +201,10 @@ func ExecWithWriters(homeDir string, args []string, stdout, stderr io.Writer) er
 	return cmd.Run()
 }
 
-func Fetch(homeDir string, version, flavor string, flavorVersion int, ms *api.Manifest) (*api.IstioDistribution, error) {
-	d, err := processFetchParams(version, flavor, flavorVersion, ms)
-	if err != nil {
-		return nil, err
-	}
-
+func Fetch(homeDir string, d *api.IstioDistribution, ms *api.Manifest) (*api.IstioDistribution, error) {
 	return d, fetch(homeDir, d, ms)
```

Comment:
put `fetch`'s logic directly here. I don't think we need to have separate `Fetch` and `fetch`.